src/adapter/src/coord.rs

```diff
@@ -363,13 +368,46 @@ pub enum TargetCluster {
     Active,
 }
 
-/// A struct to hold information on how to determine if a plan may have changed and how to re-plan
-/// it if so.
+/// A struct to hold information about the validity of peeks and if they should be abandoned after
+/// doing work off of the Coordinator thread.
 #[derive(Debug)]
-pub struct TransientPlan {
+pub struct PeekValidity {
```

Comment:
It's somewhat unfortunate that this now only works for peeks and no other plan, but I guess there's no way around this for now.